qemu/tests/cfg/driver_load_stress.cfg

```diff
@@ -117,11 +117,11 @@
             run_bgstress = rng_bat
             target_process = random\w*.exe
             session_cmd_timeout = 240
-            read_rng_cmd  = "WIN_UTILS:\random_%PROCESSOR_ARCHITECTURE%.exe"
+            read_rng_cmd  = "c:\random_%PROCESSOR_ARCHITECTURE%.exe"
             rng_data_rex = "0x\w"
             driver_id_pattern = "(.*?):.*?VirtIO RNG Device"
             during_bg_test:
-                run_bgstress = for /l %i in (1, 1, 2000) do WIN_UTILS:\random_%PROCESSOR_ARCHITECTURE%.exe &
+                run_bgstress = for /l %i in (1, 1, 2000) do c:\random_%PROCESSOR_ARCHITECTURE%.exe &
```

Comment:
`run_bgstress = for /l %i in (1, 1, 2000) do ${read_rng_cmd} &`

Reply:
May I ask why you insist to use read_rng_cmd as c:\randome.exe?
read_rng_cmd is the cmd we plan to run random util tool, we can run single random.exe or run a loop of it.
c:\random.exe is just the destination I want to copy it to.
If you want a variant to replace the string, rng_dst is better, I did not use it as I think use a default value is enough.

Reply:
> May I ask why you insist to use read_rng_cmd as c:\randome.exe?
> read_rng_cmd is the cmd we plan to run random util tool, we can run single random.exe or run a loop of it.
> c:\random.exe is just the destination I want to copy it to.
> If you want a variant to replace the string, rng_dst is better, I did not use it as I think use a default value is enough.

You are right that `read_rng_cmd` can not represent the file that we need to check in a literal sense. Before, I suggest to use `read_rng_cmd` represent the file just because i thought it is same.  If keep the current `read_rng_cmd`, add a new parameter for the dst file is ok too.